extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/handler/BoseSoundTouchHandler.java

```diff
@@ -76,6 +82,8 @@
     private CommandExecutor commandExecutor;
 
     private PresetContainer presetContainer;
+    
+    private final Counter missedPongsCount = new Counter(MAX_MISSED_PONGS_COUNT);

```

Comment:
I find it rather irritating that the "missed pongs" counter counts backwards. If it has value "2", the natural interpretation would imho be that we are missing 2 pongs already.
Wouldn't a simple `int` fully do the job here, starting at 0 and increasing it on every miss?

Reply:
I agree that a simple `int` would do the job and fit our purposes better.